distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Case_Sensitivity.enso

```diff
@@ -3,6 +3,11 @@ import project.Data.Locale.Locale
 polyglot java import org.enso.base.text.TextFoldingStrategy
 
 type Case_Sensitivity
+    ## Use the default case-sensitivity for the environment.
+       For in-memory operations, will be case sensitive.
+       For database operations, will follow the collation default.
```

Comment:
```suggestion
       For in-memory operations, it will be case sensitive.
       For database operations, it will follow the collation default.
```

Shouldn't we add the 'object' in there? I'm not 100% certain about my English grammar, but as far as I remember not including the object is not really something done in formal English, is it?